config/api_urls.py

```diff
@@ -5,4 +5,5 @@
 urlpatterns = [
     path("brands/", include("apps.brands.urls")),
     path("", include("apps.users.urls")),
```

Comment:
as a conversion always make sure the path with "" the last on the urlpatterns list

Reply:
> as a conversion always make sure the path with "" the last on the urlpatterns list

ohk
noted

Reply:
> as a conversion always make sure the path with "" the last on the urlpatterns list

will make the changes right away